tin/tests/create_users.py

```diff
@@ -0,0 +1,47 @@
+from ..apps.users.models import User
```

Comment:
`get_user_model`?

Reply:
Oops, this is a relic from the old `create_debug_users.py`